framework/Models/PostProcessors/Factory.py

```diff
@@ -36,6 +36,7 @@
 from .ParetoFrontierPostProcessor import ParetoFrontier
 from .EconomicRatio import EconomicRatio
 from .Validation import Validation
+from .validationAlgorithms.Probabilistic import Probabilistic
```

Comment:
should this be from .ValidationAlgorithms.Probabilistic import Probabilistic

Reply:
Hm, we would have to rename validationAlgorithms to ValidationAlgorithms (currently in RAVEN it is validationAlgorithms).

Reply:
resolved

Reply:
I have renamed it to ValidationAlgorithms